{"sid": "c1084a34c4f944e99c9a434fe9417001", "uid": null, "qid": "9d0d7e7377d041108dbae2dea5135c67", "rid": null, "bars": {"default": {"percent": "100", "remaining": "0"}}, "messages": [{"type": "ERROR", "content": "process - failed executing: [9d0d7e7377d041108dbae2dea5135c67]\nTraceback (most recent call last):\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/action_dispatcher.py\", line 139, in process\n    output = self.workerContext.execution.execute(data, self.ray, self.state, update)\n  File \"/pysdk/.venv/lib/python3.8/site-packages/openfabric_pysdk/app/execution/application_interface.py\", line 66, in execute\n    self.execution_function(model)\n  File \"/openfabric/app/main.py\", line 92, in execute\n    agent = Agent('gemini',\"TEST_USER\",session_id ,generator)\n  File \"/openfabric/app/Agent/agent.py\", line 38, in __init__\n    self.init_baseLLM(baseLLM)\n  File \"/openfabric/app/Agent/agent.py\", line 43, in init_baseLLM\n    history = Agent._sessionsManager.get_session_history(self.session_data.session_id)\n  File \"/openfabric/app/Agent/agent.py\", line 282, in get_session_history\n    return cls.db.get_conversation_history(session_id=session_id)\n  File \"/openfabric/app/Agent/Storage/VectorDB.py\", line 32, in get_conversation_history\n    session_history= cls._sessionsStore.get_session_history(session_id=session_id)\n  File \"/openfabric/app/Agent/Storage/VectorDB.py\", line 52, in get_session_history\n    collection = cls.db['sessions']\nAttributeError: type object 'SessionsStore' has no attribute 'db'\n", "created_at": "2025-06-30T10:08:12.515557"}], "status": "FAILED", "created_at": "2025-06-30T10:08:12.348758", "updated_at": "2025-06-30T10:08:12.515577", "finished": true}